services/masterdata/src/lib/core/utils/misc.functions.ts

```diff
@@ -0,0 +1,3 @@
+export function removeTrailingCharsFromUri(uri: string): string {
```

Comment:
I guest this can be removed if you use nswag to generate the client context (since you're not constructing urls anymore).

Reply:
i don't use nswag for masterdata (maybe i will refactor code to use nswag in a future update/version).